backend/spec/controllers/internal/companies/invoices_controller_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+RSpec.describe Internal::Companies::InvoicesController do
+  let(:company) { create(:company) }
+  let(:company_administrator) { create(:company_administrator, company:) }
+  let(:contractor) { create(:company_worker, company:) }
+  let(:user) { contractor.user }
+
+  before do
+    allow(controller).to receive(:current_context) do
+      Current.user = user
+      Current.company = company
+      Current.company_worker = contractor
+      CurrentContext.new(user:, company:)
+    end
+  end
+
+  describe "POST #create" do
+    it "returns success: true on valid create with 200 JSON" do
```

Comment:
The test says 200, but the assertion is against 200. You should make the test name generic so this does not happen. For example,

```suggestion
    it "succeeds with valid data" do
```